OrangeTree_lunch.rb

```diff
@@ -0,0 +1,137 @@
+# Group project -- Carly, Brandi, Lila
+
+require "colorize"
+
+# Define class
+class OrangeTree
+	attr_reader :height, :age, :orange_count, :death_age
```

Comment:
Do you need to expose all of these variables? It doesn't look like you read any of these variables from outside this class.